services/doc-engine/src/compare.py

```
@@ -0,0 +1,149 @@
+            "_blank",
+            0,
+            (
+                property_value("Hidden", False),
+                property_value("ReadOnly", read_only),
+                property_value("ShowTrackedChanges", True),
+            ),
+        )
+    except Exception as error:
+        raise UnreadableSource(
+            f"LibreOffice could not open the {label} Word file"
+        ) from error
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)